packages/intrepid/matlab/intrelab/CMakeLists.txt

```diff
@@ -39,159 +55,20 @@ endif()
 
 FIND_PACKAGE(Trilinos REQUIRED)
```

Comment:
I would suggest that you pass in the exact Trilinos packages that Intrelab needs with:

```
find_package(Trilinos COMPONETS <pkg1> <plg2> ... REQUIRED)
```

so that if someone tries to configure this CMake project against an install of Trilinos without the necessary packages, then the `find_package(Trilinos ... REQUIRED)` command will error out and tell them what Trilinos packages (i.e. components) are missing.  Otherwise, you will not see this mistake until build time at which point would confuse whoever is trying to builds these MEX files.